src/Test/L0/Plugin/ChunkerTests.cs

```diff
@@ -27,12 +27,12 @@ public void ChunkerIsStable(int byteCount, string expectedHash)
             var bytes = new byte[byteCount];
             FillBufferWithTestContent(seed: 0, bytes);
 
-            using (var hasher = new DedupNodeHashAlgorithm())
-            {
-                hasher.ComputeHash(bytes, 0, bytes.Length);
-                var node = hasher.GetNode();
-                Assert.Equal<string>(expectedHash, node.Hash.ToHex());
-            }
+            // using (var hasher = new DedupNodeHashAlgorithm())
```

Comment:
If we are commenting this out, should we also comment out the whole test?

Also we should leave a comment as to why it's commented out (even though we don't expect it to be disabled for long)

Reply:
Good calls, updated accordingly